docs/recipes/StayingWithExpo.md

```diff
@@ -1,29 +1,34 @@
 ---
-title: Staying With Expo
+title: Prepping Ignite for EAS Build
```

Comment:
It might be good to rename the file to match the updated post to avoid confusion